api/test/wfl/integration/modules/covid_test.clj

```diff
@@ -158,4 +162,5 @@
                (fn [] (seq (firecloud/list-entities workspace "flowcell"))))]
           (is (= name flowcell-id) "The test entity was not created"))))))
 
-(test-vars [#'test-update-terra-workspace-sink])
+(test-vars [#'test-update-terra-executor
```

Comment:
I left this in by mistake!

Reply:
Ahh!  Was going to ask about it, will remove.